src/components/NewExperiment.vue

```diff
@@ -738,13 +773,27 @@
                 </template>
               </v-autocomplete-extended>
               <v-radio-group v-model="selectedTableKey">
-                <v-radio
-                  v-for="(table, key) in tables"
-                  :key="key"
-                  :label="table.name"
-                  :value="key"
-                  color="primary"
-                ></v-radio>
+                <div v-for="(table, key) in tables" :key="key">
+                  <v-layout>
+                    <v-radio
+                      :label="table.name"
+                      :value="key"
+                      color="primary"
+                      class="mb-1"
+                    ></v-radio>
+                    <v-icon
+                      v-if="
+                        isTableValid[table.value] &&
```

Comment:
You can replace `isTableValid[table.value]` with `isTableValid[key]`

or even move isTableValid into tables[*].isValid and use `table.isValid`

Reply:
👍